### ISSUE TITLE
Build fails - ol.FeatureOverlay

### ISSUE THREAD

**Issue by petzlux** [Geoportail-Luxembourg/geoportailv3#552]
My build on branch #507 fails:

Is this connected to issue #544 ? 

````
wepa@workshop3-local:~/geoportailv3 (petzlux_location_info $)$ make -f wepa.mk serve
./node_modules/openlayers/node_modules/.bin/closure-util build build.json .build/build.js
info closure-util Reading build config
info closure-util Getting Closure dependencies
ERR! closure-util Unsatisfied dependency "ol.FeatureOverlay" in script: /home/wepa/geoportailv3/node_modules/ngeo/src/ol-ext/interaction/measure.js
make: *** [.build/build.js] Error 1
````

**Comment by rmichaelis** [comment]
Yes this is related but this is temporarily fixed here #541

**Comment by petzlux** [comment]
okay will rebase with this patch
